src/core/common/message.hpp

```diff
@@ -443,23 +443,25 @@ class Message : public Buffer
     int CopyTo(uint16_t aSourceOffset, uint16_t aDestinationOffset, uint16_t aLength, Message &aMessage) const;
 
     /**
-     * This method creates a copy of the current Message. It allocates the new one
-     * from the same Message Poll as the original Message and copies @p aLength octets of a payload.
+     * This method creates a copy of the current Message. It allocates the new one from the same message pool as the
```

Comment:
The "brief" should be on its own line.  More detailed description should be below.